core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java

```diff
@@ -299,4 +300,18 @@ static String convertUnsatisfiedConditionExceptionMessageForUpdate(
     }
     return message;
   }
+
+  /**
+   * Returns the next `tx_version` based on the current value.
+   *
+   * @param currentTxVersion The current `tx_version`, if it exists, or null otherwise.
+   * @return The next `tx_version`.
+   */
+  public static int getNextTxVersion(@Nullable Integer currentTxVersion) {
```

Comment:
This logic how to determine the next Tx version may be reused by an extension module injected via class loader.